core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala

```diff
@@ -759,6 +760,8 @@ object JsonCodec {
   implicit val azureDiskNameEncoder: Encoder[AzureDiskName] = Encoder.encodeString.contramap(_.value)
   implicit val relayNamespaceEncoder: Encoder[RelayNamespace] = Encoder.encodeString.contramap(_.value)
   implicit val aksClusterNameEncoder: Encoder[AKSClusterName] = Encoder.encodeString.contramap(_.value)
+  implicit val aksClusterEncoder: Encoder[AKSCluster] =
+    Encoder.forProduct2("name", "tags")(x => (x.name, x.tags))
```

Comment:
are we thinking that long term there may be more tags that are cluster specific that we may leverage to change behavior vs just adding a single VPA enabled vs not property?

I do like that cluster info (name and vpa status) is together, but just wondering if having the assumption we are handling more then one tag here may end up being confusing in the future?

Reply:
It's a good point -- no real strong opinions either way, but I could see the general use of tags on AKS to be useful for more things in the future. Things like: different nodepool configurations, private endpoints, different AKS preview features, etc.

Reply:
Yeah, after talking with David I think it makes more sense to leave room for other tags instead of just a bool since that's now the preferred way of info passing for WSM. I'm not sure what's confusing? tags is a Map of [string, bool] so the tag names will be explicit

Reply:
confusing as in what are all the possible tag values that can be in there and how  each one of them may be used (since the code that generates the tags lives in a different repo). Maybe a helpful comment and link to where the tags are generated can be all that is needed :)